contracts/AdvancedProviders/DelayVaultProvider/HoldersSum.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../../SimpleProviders/Provider/ProviderModifiers.sol";
+import "./MigratorV1/IDelayVaultProvider.sol";
+import "./MigratorV1/IMigrator.sol";
+
+abstract contract HoldersSum is ProviderModifiers, IDelayVaultData {
+    //this is only the delta
+    //the amount is the amount of the pool
+    // params[0] = startTimeDelta (empty for DealProvider)
+    // params[1] = endTimeDelta (only for TimedLockDealProvider)
+    event VaultValueChanged(address indexed Token, address indexed Owner, uint256 Amount);
+    mapping(address => uint256) public UserToAmount; //Each user got total amount
+    mapping(address => uint8) public UserToType; //Each user got type, can go up. wjem withdraw to 0, its reset
```

Comment:
Ownership should be handled by `LockedDealNFT`, Not sure if DelayVault needs to know the owner.